argilla/src/argilla/settings/_io/_hub.py

```diff
@@ -223,24 +229,24 @@ def _define_settings_from_features(
         else:
             warnings.warn(f"Feature '{name}' has an unsupported type. Skipping. Feature type: {feature_type}")
 
-    mapping = {
-        key: value[0] if isinstance(value, list) and len(value) == 1 else tuple(value) for key, value in mapping.items()
-    }
-
     if not questions:
-        questions.append(TextQuestion(name="comment", required=True))
+        questions.append(LabelQuestion(name="quality", required=True, labels=["good", "bad"]))
```

Comment:
Just discussed with @frascuchon and give it another thought and maybe rating with 1 to 5 is even more general and widely applicable, what do you think @burtenshaw ? This way if I want to start "categorizing" the dataset at least I have a pretty standard likert-scale for "quality"